DESCRIPTION

```diff
@@ -22,7 +22,12 @@ Imports:
 	ParallelLogger (>= 1.1.1),
 	survival,
 	plyr,
-	readr
+	readr,
+	magrittr,
+	dplyr,
+	purrr,
+	snakecase,
```

Comment:
Drop this dependency as discussed

Reply:
Will replace with new function in SqlRender here
https://github.com/OHDSI/SqlRender/commit/7bbdd3b8a3e4be9107f3f56056a0ab12987871be